src/main/java/backend/UpdateController.java

```diff
@@ -93,30 +93,31 @@ private void openRepositoriesInFilters(List<FilterExpression> filterExprs) {
      * @param filterExprs Filter expressions to process.
      * @return Filter expressions and their corresponding issues after filtering and sorting.
      */
-    private Map<FilterExpression, List<TurboIssue>> filterAndSort(List<FilterExpression> filterExprs) {
+    private Map<FilterExpression, List<TurboIssue>> processFilter(List<FilterExpression> filterExprs) {
         MultiModel models = logic.getModels();
         List<TurboIssue> allModelIssues = models.getIssues();
 
-        Map<FilterExpression, List<TurboIssue>> filteredAndSorted = new HashMap<>();
+        Map<FilterExpression, List<TurboIssue>> filteredSortedAndCounted = new HashMap<>();
 
         filterExprs.forEach(filterExpr -> {
-            List<TurboIssue> filterAndSortedExpression = filteredAndSorted.get(filterExpr);
+            List<TurboIssue> filterAndSortedExpression = filteredSortedAndCounted.get(filterExpr);
 
             if (filterAndSortedExpression == null) { // If it already exists, no need to filter anymore
                 boolean hasUpdatedQualifier = Qualifier.hasUpdatedQualifier(filterExpr);
 
                 FilterExpression filterExprNoAlias = Qualifier.replaceMilestoneAliases(models, filterExpr);
 
-                List<TurboIssue> filteredAndSortedIssues = allModelIssues.stream()
+                List<TurboIssue> filteredSortedAndCountedIssues = allModelIssues.stream()
                         .filter(issue -> Qualifier.process(models, filterExprNoAlias, issue))
                         .sorted(determineComparator(filterExprNoAlias, hasUpdatedQualifier))
```

Comment:
That's a method from the new streams API. I think the convention is that verbs work on one element at a time, but adjectives work on the stream as a whole.